### PR TITLE
Consolidating CSS Files For Issue #49

### PR DESCRIPTION
**Pull request by lRolwes**
Initial Problem : CSS was scattered among many files and very hard to make sense of

End result: CSS is consolidates between inline css and App.css

What changed:
- Removed SystemDetails.css
- removed index.css
- Changed SystemDetails.js to import App.css instead of SystemDetails.css
- Moved contents of SystemsDetails.css into App.js
- imported app.css into exploreSystems
- removed page2.css
- swapped table.css with app.css in CitationsTable.js
- added  app.css to CriticalPointsTable.js
- replaced table.css with app.css iSystemDetails
- Deleted table.css

### CODE REVIEW

**Comment by Evan-Richmond** [comment]
Title needs changed before merging

**Review by AJelliss — APPROVED** [review]
looking good to me

File: Frontend/src/components/FunctionDetail/CitationsTable.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import './table.css'
+import './../../App.css'
```
Comment (AbhilashKotha) [inline Frontend/src/components/FunctionDetail/CitationsTable.js:1]: You do not have to import the App.css in all the files, just importing it in the index.js file should be sufficient.

**Review by AbhilashKotha — CHANGES_REQUESTED** [review]
You do not have to import the App.css in all the files, just importing it in the index.js file should be sufficient.

File: Frontend/src/components/FunctionDetail/CitationsTable.js (hunk 1 above)
Comment (lRolwes) [reply Frontend/src/components/FunctionDetail/CitationsTable.js]: I have updated accordingly

**Review by AbhilashKotha — APPROVED** [review]
lgtm;

**Pull request merged**